docs/source/tools/visual.rst

```diff
@@ -9,4 +9,16 @@ You can generate visual graphs for the contracts in your DAML project. To do thi
 1. Install `Graphviz <http://www.graphviz.org/download/>`_.
 2. Generate a DAR from your project by running ``daml build``.
 3. Generate a `dot file <https://en.wikipedia.org/wiki/DOT_(graph_description_language)>`_ from that DAR by running ``daml damlc visual <path_to_project>/dist/<project_name.dar> --dot <project_name>.dot``
-4. Generate the visual graph with Graphviz by running ``dot -Tpng <project_name>.dot > <project_name>.dot.png``
\ No newline at end of file
+4. Generate the visual graph with Graphviz by running ``dot -Tpng <project_name>.dot > <project_name>.png``.
+
+
+Visual for Quickstart
+*********************
+1. Generate the dar using ``daml build``.
```

Comment:
Can you add a sentence or two of introduction here, explaining what this is? Essentially saying if you've run through the quickstart guide (or even potentially just cloned it, that might do), you can generate a visual like this. And don't forget to link to the quickstart guide (using `:doc:` - you can search through the docs folder to see examples of how that works)

Reply:
You also have an assumption that they've installed Graphviz, might want to be explicit about that.